src/interfaces/particles/spin_pol.jl

```diff
@@ -179,13 +179,39 @@ const PolY = PolarizationY
 Base.show(io::IO, ::PolY) = print(io, "y-polarized")
 
 """
-    multiplicity(spin_or_pol)
+    SyncedPolarization{N::Int} <: AbstractIndefinitePolarization
 
-Return the number of spins or polarizations respresented by `spin_or_pol`, e.g. `multiplicity(SpinUp()) == 1`, but `multiplicity(AllSpin()) = 2`.
+An indefinite polarization type, indicating that multiple particles have a synced polarization.
+Two polarizations are considered synced when they have the same value for `N`. This means that
+the resulting multiplicity will be 2 total for all particles with the same `SyncedPolarization`.
 
+Having a single `SyncedPolarization{N}` in a process is legal. In this case, it behaves just
+like an [`AllPolarization`](@ref) would.
+
+See also: [`multiplicity`](@ref)
+"""
+struct SyncedPolarization{N} <: AbstractIndefinitePolarization
+    function SyncedPolarization{N}() where {N}
```

Comment:
Is it really necessary to expose the type parameter in the constructor, wouldn't it be more convenient to have a standard call constructor like 

```julia 
function SyncedPolarization(N::Int)
    return new{N}()
end
```
Then the type assert is not necessary. If you need the constructor like it's implemented, the constructor above should conveniently added as well.

Reply:
You're right, that is much nicer and requires fewer brackets that I definitely didn't forget about ten times while writing this code